ecl/sent2WordListTest.ecl

```diff
@@ -0,0 +1,55 @@
+/*##############################################################################
```

Comment:
I think the test case name is a bit misleading and/or confusing, because it 'sent' nothing. I feel the 'sentence2WordListTest.ecl' would be better.